android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -848,6 +848,17 @@ public void setZoom(double zoom) {
         }
     }
 
+    public void setAntiAliasing(boolean enableAntialiasing) throws PDFNetException {
+        if (getPdfViewCtrl() != null) {
+            try{
```

Comment:
always format your code

Reply:
yes there is one note though I have enabled format on save so the whole document gets reformatted. Is that ok?